sys-process/oomd/oomd-0.5.0-r1.ebuild

```
@@ -0,0 +1,44 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+DESCRIPTION="oomd is an userspace Out-Of-Memory (OOM) killer for Linux systems."
+HOMEPAGE="https://github.com/facebookincubator/oomd"
+SRC_URI="https://github.com/facebookincubator/oomd/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64"
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="systemd"
+DEPEND="dev-libs/jsoncpp:=
+	systemd? ( sys-apps/systemd )"
```

... and if it does link against libsystemd.so, add 
```suggestion
	systemd? ( sys-apps/systemd:= )"
```
here.